go/api/response_handlers.go

```diff
@@ -31,6 +32,72 @@ func handleStringOrNullResponse(response *C.struct_CommandResponse) string {
 	return handleStringResponse(response)
 }
 
+// handleIntegerResponse handles the response from a command that returns an integer.
+func handleIntegerResponse(response *C.struct_CommandResponse) int64 {
```

Comment:
handleLongResponse has the same functionality.

Reply:
@janhavigupta007 removed `handleIntegerResponse` func.